Agreements/Internal/TCWEL_28.txt

```diff
@@ -0,0 +1,42 @@
+COPX:EGKK:*:BEDFO:*:*:London TC WELIN:London S28:21000:*:^BEDFO
+
+COPX:EGKK:*:WELIN:*:*:London TC WELIN:London S28:21000:*:^WELIN
+
+COPX:EGLL:*:TIMPO:*:*:London TC WELIN:London S28:19000:*:TIMPO
+COPX:EGLL:*:TNT:*:*:London TC WELIN:London S28:19000:*:TNT35
+
+COPX:EGWU:*:TIMPO:*:*:London TC WELIN:London S28:19000:*:TIMPO
+COPX:EGWU:*:TNT:*:*:London TC WELIN:London S28:19000:*:TNT35
+
+COPX:EGLC:*:TIMPO:*:*:London TC WELIN:London S28:19000:*:TIMPO
+COPX:EGLC:*:PIPIN:*:*:London TC WELIN:London S28:19000:*:PIPIN
+COPX:EGLC:*:TNT:*:*:London TC WELIN:London S28:19000:*:TNT35
+
+COPX:EGKB:*:TIMPO:*:*:London TC WELIN:London S28:19000:*:TIMPO
+COPX:EGKB:*:PIPIN:*:*:London TC WELIN:London S28:19000:*:PIPIN
+COPX:EGKB:*:TNT:*:*:London TC WELIN:London S28:19000:*:TNT35
+
+COPX:EGMC:*:TIMPO:*:*:London TC WELIN:London S28:19000:*:TIMPO
+COPX:EGMC:*:PIPIN:*:*:London TC WELIN:London S28:19000:*:PIPIN
+COPX:EGMC:*:TNT:*:*:London TC WELIN:London S28:19000:*:TNT35
+
+COPX:EGTO:*:TIMPO:*:*:London TC WELIN:London S28:19000:*:TIMPO
+COPX:EGTO:*:PIPIN:*:*:London TC WELIN:London S28:19000:*:PIPIN
+COPX:EGTO:*:TNT:*:*:London TC WELIN:London S28:19000:*:TNT35
+
+COPX:EGsS:*:TIMPO:*:*:London TC WELIN:London S28:19000:*:TIMPO
+COPX:EGSS:*:PIPIN:*:*:London TC WELIN:London S28:19000:*:PIPIN
+COPX:EGSS:*:TNT:*:*:London TC WELIN:London S28:19000:*:TNT35
+
+COPX:EGSC:*:TIMPO:*:*:London TC WELIN:London S28:19000:*:TIMPO
+COPX:EGSC:*:PIPIN:*:*:London TC WELIN:London S28:19000:*:PIPIN
+COPX:EGSC:*:TNT:*:*:London TC WELIN:London S28:19000:*:TNT35
+
+COPX:EGGW:*:TIMPO:*:*:London TC WELIN:London S28:19000:*:TIMPO
+COPX:EGGW:*:PIPIN:*:*:London TC WELIN:London S28:19000:*:PIPIN
+COPX:EGGW:*:TNT:*:*:London TC WELIN:London S28:19000:*:TNT35
+
+COPX:EGHI:*:LESTA:*:*:London TC WELIN:London S28:21000:*:LESTA
```

Comment:
Climbing?

Reply:
According to what I have, level by  agreements also